### DISCUSSION TITLE
Reverting back to Flsun version of Klipper

### DISCUSSION THREAD

**Discussion by arshish1612** [Guilouz/Klipper-Flsun-Speeder-Pad#70] [General]
Hi,

I'm about to receive my V400 this week and I'm about to go down the path of moving to the official build. 

I've been reading the documentation carefully and I think I understand what needs to be done. 

Is there an easy way to revert back to the FLSun build incase I mess up?

**Comment by arshish1612** [comment]
Actually... Should I jump right in and move to an official Klipper build or should I spend some time with the FLSUN build and get to know the printer better first?

Also are the calibration steps with the official Build and the FLSUN build the same?

**Reply by 808mp5** [reply]
If i could start all over and you plan to move to Guilouz build... don't even mess with the FLSUN build.. it will only cause more confusion as a lot of it is different.  You will have to re-learn most of it.. so just learn it once is what i would suggest
